spec/models/prison_spec.rb

```diff
@@ -122,4 +120,20 @@
       end
     end
   end
+
+  describe '#allocations_for_pom' do
+    it 'returns all AllocatedOffenders in this prison for the POM with the given nomis_staff_id' do
+      prison = FactoryBot.create :prison
+      nomis_staff_id = 'STAFF1'
+      expected_allocations = double(:expected_allocations)
+      # This crap codebase forces us to stub out methods on the class we're testing - this is awful practice and a "test
+      # smell" indicating badly drawn boundaries between components. But refactoring the whole thing is not practical so
+      # we are forced to use this bad practice.
+      alloc_relation = double :allocation_relation, for_pom: []
+      allow(prison).to receive(:allocations).and_return(alloc_relation)
+      allow(alloc_relation).to receive(:for_pom).with(nomis_staff_id).and_return(expected_allocations)
```

Comment:
If I had a test with multiple expectations the I'd do that, yes.

Reply:
Ah I remember why I didn't put those in let/before:

Normally, yeah, we pull out common stuff into let and before statements. But because the rest of the test is a huge pile of badly structured shit, I'm basically ignoring everything and treating this as a separate little test. Maybe that's a bad thing because I'm not refactoring the rest of the test, but I don't really believe in refactoring tests since then you are never sure if they'll break if the condition they're testing fails.